docs/releases/ez_platform_v1.11.0.md

```diff
@@ -0,0 +1,103 @@
+
+If you are looking for the Long Term Support (LTS) release, see [https://ezplatform.com/Blog/Long-Term-Support-is-Here](https://ezplatform.com/Blog/Long-Term-Support-is-Here)
+
+## Notable changes since v1.10.0
+
+### eZ Platform
+
+#### Improved way of writing Field Type gateways
+
+[EZP-26885](https://jira.ez.no/browse/EZP-26885): you now have access to the Doctrine connection instead of
+the Zeta Components Database connection-like object which has been exposed to Field Types until now.
+The former way will be removed in a future major version.
+
+#### Content Type limitation for Relation (single) field
+
+[EZP-24800](https://jira.ez.no/browse/EZP-24800): you can now specify a Content Type limitation for the Relation field,
+just like with the Relation List field. This enables you to limit what kind of relations Editors can select also on singular relation fields.
+
+![Adding a new Relation (single) Field with allowed Content Types](img\relation_single_allowed_cts.png)
+
+This has been made possible by initial legacy contribution from [@peterkeung](https://github.com/peterkeung), and [@slaci](https://github.com/slaci) who ported this feature over to eZ Platform so that both could go in.
+
+#### API endpoint for removing translations
+
+[EZP-27417](https://jira.ez.no/browse/EZP-27417) provides an API endpoint to remove a given translation completely from a Content item.
+
+### eZ Platform Enterprise Edition
+
+#### Collection block
+
+New Collection block is available in the Landing Page editor.
+It enables you to manually select a set of Content items to be displayed.
+
+![Collection block options with three Content items selected](img/collection_block.png)
+
+!!! note
+
+    To enable adding content to a Collection block in a clean installation,
+    you need to configure the views for the block and define which Content Types can be embedded in it.
+    See [block templates](../guide/extending_ez_platform_ui.md#block-templates) for more information and an example.
+
+#### RecommendationBundle adapted for YooChoose v2
+
+In the RecommendationBundle, the id generation of a visitor was changed to use a persistent cookie value
+instead of a new one each time a visitor arrives at the site.
+
+Fetching recommendations was also refactored to use the v2 of the recommendation API.
+With this step the clickrecommended event now includes detailed feedback information about how recommendations were generated.
+This is very important for the analysis of statistics to measure the performance of recommendations.
+
+See [EZEE-1611](https://jira.ez.no/browse/EZEE-1611) for details.
+
+#### Official Enterprise Support for Legacy Bridge
+
+Starting with this release we are going to officially support an alternative *(and perhaps simpler)* way to gradually migrate
+from eZ Publish to eZ Platform. From now on, also as an Enterprise user, you can use **Legacy Bridge**.
```

Comment:
You mean the source-view?

source weird newline effect:
http://i.share.pho.to/ec0e86c4_o.png
(even here: http://i.share.pho.to/54f94f38_o.png)

and rendered MD on Github: http://i.share.pho.to/2032c1fc_o.png